include/fluent-bit/flb_aws_credentials.h

```diff
@@ -156,6 +163,42 @@ struct flb_aws_provider *flb_standard_chain_provider_create(struct flb_config
                                                             flb_aws_client_generator
                                                             *generator);
 
+/* Provide base configuration options for managed chain */
+#define FLB_AWS_CREDENTIAL_BASE_CONFIG_MAP(prefix)                              \
+    {                                                                           \
+     FLB_CONFIG_MAP_STR, prefix "region", NULL,                                 \
+     0, FLB_FALSE, 0,                                                           \
+     "AWS region of your service"                                               \
+    },                                                                          \
+    {                                                                           \
+     FLB_CONFIG_MAP_STR, prefix "sts_endpoint", NULL,                           \
+     0, FLB_FALSE, 0,                                                           \
+     "Custom endpoint for the AWS STS API, used with the AWS_Role_ARN option"   \
+    },                                                                          \
+    {                                                                           \
+     FLB_CONFIG_MAP_STR, prefix "role_arn", NULL,                               \
+     0, FLB_FALSE, 0,                                                           \
+     "ARN of an IAM role to assume (ex. for cross account access)"              \
```

Comment:
Change to `AWS IAM Role` so align with line 186 to reduce confusion.

Reply:
Aligning with value in Cloudwatch, Firehose, Kinesis, and S3. Future plans to potentially unify the code for credential provider setup, need to keep this 
```
ARN of an IAM role to assume 
```